### ISSUE TITLE
Create endpoint that returns the list of classes we have controlled terms for

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/api#176]
Path would be at `/attributes/`

Example response: `["nb:Diagnosis", "nb:Assessment", "nb:Sex", "nb:SubjectGroup", "nb:Image"]`.
Response should probably include only subclasses of `nb:ControlledTerm` (see also https://github.com/neurobagel/api/blob/main/vocab/nb_vocab.ttl) and omit classes whose instances are just literals, like `"nb:Age"`.

- [x] add test that API response is a list
- [x] replace namespace URL in response with prefixes for easier use by downstream tools

**Comment by surchs** [comment]
I think unless we want to keep and update a list of these classes by hand, addressing this will require changes in the graph so that all the classes have some easily searchable property, e.g. they are all subclasses off nb:ControlledTerm or something.

Two quick thoughts:
- This new property could be added to every dataset jsonld by the cli (and then also updated in the output data model)
- Better but more work would be to have our own mini vocabulary with all possible classes and their subclass relationship to eg nb:ControlledTerm. This could then get merged into the graph once, just like we'll eventually have to do for cogatlas terms etc

**Comment by alyssadai** [comment]
The second option sounds reasonable to me @surchs. I'm assuming in that case we would provide a JSON-LD file containing the vocabulary in our repo, which we would instruct users to add to their graph upon deployment, right?

**Comment by surchs** [comment]
> we would instruct users to add to their graph upon deployment, right

something like that yes, or make it part of our automatic deployment. 
 
> we would provide a JSON-LD file

Yeah, I think that makes sense. We can build that by hand for now.

**Comment by alyssadai** [comment]
Blocked by https://github.com/neurobagel/api/issues/182
